qiita_db/util.py

```diff
@@ -1651,22 +1649,22 @@ def get_artifacts_information(artifact_ids, only_biom=True):
                     'deprecated': cmd.software.deprecated}
 
             # now let's get the actual artifacts
-            ts = {}
+            ts = {None: []}
```

Comment:
`None` for a key is weird, why is this needed?

Reply:
ts is a cache (prep id : target subfragment) so we don't have to query multiple times the target subfragment for a prep info file. Now, some artifacts (like analysis) do not have a prep info file; thus the None. It's possible to do and if/else but this was easier when first written, let me know ...

Reply:
If `None` corresponds to analyses which do not have a prep (or it doesn't make sense to have one), would it be possible to include a comment as such denoting the interpretation of the key?